XO.py

```diff
@@ -0,0 +1,157 @@
+from random import randint
+from collections import namedtuple
+
+# Size of board.
+N = 3
+
+
+class Model:
+    def __init__(self):
+        self.board = [[None for x in range(N)] for y in range(N)]
+
+    def get_board_status(self):

```

Comment:
We're not in Java, so no need for `getters` :)

Reply:
Did you mean just the name of the function or the need for the function itself? If I understood correctly, you meant that unlike Java, I can access the model class variables from the controller, so I don't need a special function to return them. I fixed it :)